qdrant-landing/content/documentation/concepts/points.md

```diff
@@ -1399,6 +1399,27 @@ client
     .get();
 ```
 
+```csharp
+await client.DeleteVectorsAsync("{collection_name}", ["text", "image"], [0, 3, 10]);
+```
+
+```go
+import (
+	"context"
+
+	"github.com/qdrant/go-client/qdrant"
+)
+
+client.DeleteVectors(context.Background(), &qdrant.DeletePointVectors{
+	CollectionName: "{collection_name}",
+	PointsSelector: qdrant.NewPointsSelector(
+		qdrant.NewIDNum(0), qdrant.NewIDNum(3)),
```

Comment:
shouldn't we add `, qdrant.NewIDNum(10)` in the Go sample as well, be align with the other languages?